src/main/java/com/odysseusinc/arachne/portal/util/SearchIndicesRefresher.java

```diff
@@ -0,0 +1,105 @@
+/*
+ *
+ * Copyright 2020 Odysseus Data Services, inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Company: Odysseus Data Services, Inc.
+ * Product Owner/Architecture: Gregory Klebanov
+ * Authors: Alexandr Cumarav, Vitaly Koulakov, Yaroslav Molodkov
+ * Created: November 04, 2020
+ */
+
+package com.odysseusinc.arachne.portal.util;
```

Comment:
why is this util package? I would place it in the Services + extract the logic from BaseAdminController.reindexSolr to here.

Reply:
It is not a service. It is single puprpose component which uses other services to trigger reindexing on the app start

Reply:
Ok, this is not a service but this is not util, so the util pkg is not a good choice.

Reply:
moved to jobs package